taskcluster/fxci_config_taskgraph/util/integration.py

```diff
@@ -41,7 +41,11 @@ def find_tasks(decision_index_path: str) -> list[dict[str, Any]]:
     tasks = []
     for task in task_graph.values():
         assert isinstance(task, dict)
-        if task.get("attributes", {}).get("unittest_variant") != "os-integration":
+        attributes = task.get("attributes", {})
+        if attributes.get("unittest_variant") != "os-integration":
+            continue
+
+        if attributes.get("test_platform", "").startswith(("android-hw", "macosx")):
```

Comment:
If we're not going to run these tasks as part of these tests, should we stop running as part of the cron task in Gecko rather than ignoring them here?

Reply:
Yeah, I fixed it there already too. But the cron task won't run again until Monday, so this unblocks us in the meantime (and provides an extra safeguard from future mistakes)